platform/lib/samples/DocumentParser.js

```diff
@@ -61,7 +61,7 @@ const BEAUTIFY_OPTIONS = {
   indent_size: 2,
   'wrap_attributes': 'force',
   //"wrap_attributes_indent_size": 4,
-  unformatted: ['noscript', 'style', 'head', 'script'],
+  unformatted: ['noscript', 'style', 'head', 'script', 'html'],
```

Comment:
This was the missing piece. Transformed ⚡ 4email examples were not correctly identified, since the there was a line break that caused the DocumentParser to identify them as regular amp.

Reply:
Good find! That must've been tricky to work out.